src/com/ppm/integration/agilesdk/connector/agilecentral/RallyIntegrationConnector_ar.properties

```diff
@@ -1,51 +1,13 @@
-ERROR_AUTHENTICATION_FAILED=\u0641\u0634\u0644\u062a \u0627\u0644\u0645\u0635\u0627\u062f\u0642\u0629
-ERROR_RELEASE_IS_NOT_STARTED=\u0627\u0644\u0625\u0635\u062f\u0627\u0631 \"{0}\" \u0644\u0645 \u064a\u0628\u062f\u0623
-ERROR_RELEASE_IS_FINISHED=\u0627\u0646\u062a\u0647\u0649 \"{0}\" \u0627\u0644\u0625\u0635\u062f\u0627\u0631
-ERROR_CANNOT_GET_CURRENT_SPRINT=\u0644\u0627 \u064a\u0645\u0643\u0646 \u0627\u0644\u062d\u0635\u0648\u0644 \u0639\u0644\u0649 \u0633\u0628\u0627\u0642 \u062d\u0627\u0644\u064d
-WARN_TM_AGM_RELEASE_NOT_ALIGNED=\u064a\u0634\u064a\u0631 \u0625\u0644\u0649 \u0623\u0646 \u0627\u0644\u0625\u0635\u062f\u0627\u0631 \u0645\u0646 AGM \u063a\u064a\u0631 \u0645\u062a\u0637\u0627\u0628\u0642 \u0645\u0639 \u0641\u062a\u0631\u0629 \u062c\u062f\u0648\u0644 PPM \u0627\u0644\u0632\u0645\u0646\u064a.
-ERROR_HTTP_CONNECTIVITY_ERROR=\u062e\u0637\u0623 \u0641\u064a \u0627\u0644\u0627\u062a\u0635\u0627\u0644 \"{0}\"
-ERROR_TIMESHEET_RETRIEVE=\u062e\u0637\u0623 \u0641\u064a \u0627\u0633\u062a\u0631\u062f\u0627\u062f \u0628\u064a\u0627\u0646\u0627\u062a \u0627\u0644\u0635\u062d\u064a\u0641\u0629 \u0627\u0644\u0632\u0645\u0646\u064a\u0629 \"{0}\"
-ERROR_ACCESS_TOKEN_RETRIEVE=\u062a\u0639\u0630\u0631 \u0627\u0644\u0648\u0635\u0648\u0644 \u0625\u0644\u0649 \u0627\u0644\u0645\u0642\u0637\u0639 \u0627\u0644\u0645\u0645\u064a\u0632
-CLIENT_ID_OR_SECRET_KEY_NOT_SET=\u0644\u0645 \u064a\u062a\u0645 \u062a\u0639\u064a\u064a\u0646 \u0645\u0639\u0631\u0641 \u0627\u0644\u0639\u0645\u064a\u0644 \u0623\u0648 \u0627\u0644\u0645\u0641\u062a\u0627\u062d \u0627\u0644\u0633\u0631\u064a
-#for create release fields
-ERROR_WORKSPACE_ID=\u062a\u0639\u0630\u0631 \u0627\u0644\u062d\u0635\u0648\u0644 \u0639\u0644\u0649 \u0645\u0639\u0631\u0641 \u0645\u0633\u0627\u062d\u0629 \u0627\u0644\u0639\u0645\u0644
-IS_CREATE_RELEASE=\u0647\u0644 \u062a\u0631\u064a\u062f \u0625\u0646\u0634\u0627\u0621 \u0625\u0635\u062f\u0627\u0631 \u062c\u062f\u064a\u062f\u061f
-CREATE_RELEASE_NAME=\u0627\u0644\u0627\u0633\u0645
-CREATE_RELEASE_DESCRIPTION=\u0627\u0644\u0648\u0635\u0641
-CREATE_RELEASE_START_TIME=\u0648\u0642\u062a \u0627\u0644\u0628\u062f\u0621
-CREATE_RELEASE_END_TIME=\u0648\u0642\u062a \u0627\u0644\u0627\u0646\u062a\u0647\u0627\u0621
-CREATE_RELEASE_SPRINT_DURATION=\u0645\u062f\u0629 \u0627\u0644\u0633\u0628\u0627\u0642
-CREATE_RELEASE_SPRINT_DURATION_UNIT=\u0648\u062d\u062f\u0629 \u0645\u062f\u0629 \u0627\u0644\u0633\u0628\u0627\u0642
-CREATE_RELEASE_DAY=\u0627\u0644\u064a\u0648\u0645
-CREATE_RELEASE_WEEK=\u0627\u0644\u0623\u0633\u0628\u0648\u0639
-NO_DATA_AVAILABLE=\u0644\u0627 \u062a\u062a\u0648\u0641\u0631 \u0628\u064a\u0627\u0646\u0627\u062a.
+USERNAME=Username
+PASSWORD=Password
+PROXY_HOST=Proxy Host
+PROXY_PORT=Proxy Port
+SUBSCRIPTION=Subscription
+WORKSPACE=\u0645\u0633\u0627\u062d\u0629 \u0627\u0644\u0639\u0645\u0644
```

Comment:
So we're giving up arabic translation but still have a translated label for Workspace? Strange!